rerun_cpp/src/rerun/recording_stream.hpp

```diff
@@ -868,34 +870,87 @@ namespace rerun {
         ///
         /// Any failures that may occur during serialization are handled with `Error::handle`.
         ///
+        /// \param entity_path The path to the recording property.
+        /// \param properties The properties of the recording.
+        /// \see `try_set_properties`
+        template <typename... Ts>
+        void set_properties_with_prefix(std::string_view entity_path, const Ts&... properties)
+            const {
+            try_set_properties_with_prefix(entity_path, properties...).handle();
+        }
+
+        /// Set the properties of a recording.
+        ///
+        /// \param entity_path The path to the recording property.
+        /// \param properties The properties of the recording.
+        /// \see `set_properties`
+        template <typename... Ts>
+        Error try_set_properties_with_prefix(std::string_view entity_path, const Ts&... properties)
+            const {
+            rr_error status = {};
+            this->log_static(
+                this->PARTITION_PROPERTIES_ENTITY_PATH + std::string(entity_path),
```

Comment:
should add a slash here so user can specify `"thing"` as path and not break things

Reply:
Ah good catch!